server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -60,7 +60,7 @@
                     <h2 class="app-card__title" data-qa="{{ moveType }}-title-{{ loopIndex }}">
                         {% if user.isReceptionUser %}
```

Comment:
need to check whether this can be removed - I think all staff should be able to see the summary?
But not confirm